### PR TITLE
Add ImagePicker functionality to Avatar component.

### PR DESCRIPTION
**Pull request by swanka**

### CODE REVIEW

File: packages/ui/src/Avatar.tsx [hunk 1]
```diff
@@ -37,12 +39,34 @@ interface AvatarProps {
   /**
    * The URL of the user's image.
    */
-  src?: string;
+  imageSrc?: string;
```
Comment (joshgachnang) [inline packages/ui/src/Avatar.tsx:42]: Any reason to change this from src?

Comment (swanka) [reply packages/ui/src/Avatar.tsx:42]: Honestly, it was just to make it more explicit but I have no strong opinion one way or the other.

Comment (joshgachnang) [reply packages/ui/src/Avatar.tsx:42]: I would lean towards keeping the prop name and not breaking the API whenever possible.

Comment (swanka) [reply packages/ui/src/Avatar.tsx:42]: You got it. I'll switch it back

**Review by joshgachnang — APPROVED** [review]
Sweet, LGTM with the `src` change!

**Pull request merged**